openstack/keystone/templates/ingress-api.yaml

```diff
@@ -66,7 +66,7 @@ metadata:
     ingress.kubernetes.io/auth-tls-verify-depth: "3"
     {{- end }}
     {{- if .Values.services.ingress.vice_president }}
```

Comment:
To reduce confusion, this key `.Values.services.ingress.vice_president` should be renamed, too

Reply:
same for line 73 below